src/components/UserInfo/UserInfo.tsx

```diff
@@ -1,5 +1,14 @@
 import React from 'react';
+import { User } from '../../types/User';
+import './UserInfo.scss';
+
+type Props = {
+  user: User
+};
+
+export const UserInfo: React.FC<Props> = ({ user }) => (
+  <a className="UserInfo" href={`mailto:${user.email}`}>
+    {user.name}
+  </a>
```

Comment:
use decomposing structure
```suggestion
export const UserInfo: React.FC<Props> = ({ user: { email, name } }) => (
  <a className="UserInfo" href={`mailto:${email}`}>
    {name}
  </a>
```

Reply:
To get rid of spread in props, followed Misha`s approach in the tutorial. Please, let me know if there is another way to do it.